pyproject.toml

```diff
@@ -66,6 +66,7 @@ optional = true
 
 
 [tool.poetry.group.docs.dependencies]
+sphinx = "~7.2.0"

```

Comment:
Our theme breaks with Sphinx 7.3 and up. That's an issue we're tracking in #273. For now fixed by pinning 7.2.